napari/layers/intensity_mixin.py

```diff
@@ -121,7 +125,9 @@ def gamma(self):
 
     @gamma.setter
     def gamma(self, value):
+        self.events.gamma(value=value)
+
+    def _on_gamma(self, value):
```

Comment:
Again, I don't understand why some attributes have "on_attr_**change**" while others don't...

Reply:
it's a typo that I'm fixing

Reply:
Good to know I'm not crazy! =)